src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,35 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int maxDiggingDepth;
```

Comment:
The field 'maxDiggingDepth' should have an access modifier. Consider using private to encapsulate the field properly.